hieradata/common.yaml

```diff
@@ -380,6 +380,9 @@ govuk::apps::specialist_publisher::enabled: true
 govuk::apps::specialist_publisher::nagios_memory_warning: 800
 govuk::apps::specialist_publisher::nagios_memory_critical: 900
 
+govuk::apps::specialist_publisher_rebuild::enabled: true
+govuk::apps::specialist_publisher_rebuild::publish_pre_production_finders: true
```

Comment:
Setting this to `true` without overriding it to `false` for Production and Staging will mean that pre-production finders will be published in all environments, because common hieradata is used as the base for all environments (I think...). The param on the class [already defaults to `false`](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/specialist_publisher_rebuild.pp#L74), so omitting this line and not removing the `true` override for dev and Integration would do what you want.